hera_cal/abscal.py

```diff
@@ -2436,11 +2436,15 @@ def custom_dly_slope_gain(self, gain_keys, antpos):
         antpos : type=dictionary, contains antenna position vectors. keys are ant integer, values are ant position vectors
         """
         if hasattr(self, '_dly_slope'):
-            # get dly slope dictionary
-            dly_slope = self.dly_slope[self._gain_keys[0][0]]
-            # turn delay slope into per-antenna complex gains, while iterating over gain_keys
-            return odict(list(map(lambda k: (k, np.exp(2j * np.pi * self.freqs.reshape(1, -1) * np.einsum("i...,i->...", dly_slope, antpos[k[0]][:2]))),
-                                  gain_keys)))
+            # form dict of delay slopes for each polarization in self._gain_keys
+            # b/c they are identical for all antennas of the same polarization
+            dly_slope_dict = {k[0][1]: self.dly_slope[k[0]] for k in self._gain_keys}
```

Comment:
Given that this is a nested list, it might be more readable if you use something more evocative than `k` for the iterator. Perhaps `for ants in self._gain_keys`?